src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -222,11 +302,12 @@ namespace OptionTags {
  * *rolled-on* to) at the beginning of an evolution. This parameter sets
  * the coordinate time at which roll-on begins.
  */
-struct GaugeHRollOnStartTime : db::SimpleTag {
+struct GaugeHRollOnStart {
```

Comment:
One last comment on these tag names: I think you originally changed these names to make them shorter to fit in options. You don’t need to do that anymore since you implement a name function.

Reply:
The names look like are back to the longer names now

Reply:
Ok, I won't insist on changing these names